scripts/participateInAuction.js

```diff
@@ -22,7 +22,7 @@ async function participateInAuction(coordinatorAddress, network) {
   const debugAuction = Debug("DEBUG-participate:" + coordinatorAddress)
   const infoAuction = Debug("INFO-participate:" + coordinatorAddress)
 
-  debugAuction("Do participate in auction")
+  debugAuction("Check if it should participate in auction")
```

Comment:
I am super glad to see this change. Sounded weird before, but I had no suggestion

Reply:
Hehe, I agree!